### PR TITLE
Docs: Hosted Authenticate Service capabilities page and sidebar

### PR DESCRIPTION
**Pull request by ZPain8464**
This PR adds a Hosted Authenticate Service capabilities page and sidebar slice. 

This is a first draft. It could probably be more informative in sections like Less time to deploy and Privacy considerations. 

Once a Hosted Authenticate URL reference page is live, it will also need to be linked here.

### CODE REVIEW

**Review by cmo-pomerium — APPROVED** [review]
I have copy reviewed. LGTM

File: content/docs/capabilities/hosted-authenticate-service.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+The Hosted Authenticate Service is available for both open-source Pomerium Core and Enterprise users.
+
+## Why use Hosted Authenticate Service
+
+The Hosted Authenticate Service offers a quicker way for users to deploy and test Pomerium.
+
+### Zero configuration
+
+Including the hosted authenticate URL in your configuration file authenticates your users against our hosted identity provider solution (we use Cognito).
```
Comment (calebdoxsey) [inline content/docs/capabilities/hosted-authenticate-service.md:21]: We will default to the hosted authenticate URL, so, something like:

By default Pomerium will use the hosted identity provider if no `authenticate_service_url` is defined.

File: content/docs/capabilities/hosted-authenticate-service.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+This means you only need to include the hosted URL in your configuration file to use Pomerium's hosted services — no [identity provider configuration](/docs/identity-providers) required.
```
Comment (calebdoxsey) [inline content/docs/capabilities/hosted-authenticate-service.md:23]:
```suggestion
This means no [authenticate service URL](/docs/reference/authenticate-service-url) or  [identity provider configuration](/docs/identity-providers) is required.
```

File: content/docs/capabilities/hosted-authenticate-service.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+### Faster proof of concept
+
+If you're testing Pomerium for the first time, run [Pomerium with Docker](/docs/quickstart) using our hosted services – you can run Pomerium Core in **under 5 minutes** with minimal setup.
+
+Current Pomerium users who are interested in our [Enterprise Console](https://www.pomerium.com/enterprise-sales/) can test out the [Docker Enterprise Quickstart](/docs/releases/enterprise/install/quickstart) using hosted services as well.
+
+## How to use the Hosted Authenticate Service
+
+1. Include the Hosted Authenticate Service URL in your configuration file
```
Comment (calebdoxsey) [inline content/docs/capabilities/hosted-authenticate-service.md:37]:
```suggestion
```

File: content/docs/capabilities/hosted-authenticate-service.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+1. Remove your IdP settings
+1. Set up a route and policy
+
+```yaml title=pomerium-config.yaml
+authenticate_service_url: https://authenticate.pomerium.app
```
Comment (calebdoxsey) [inline content/docs/capabilities/hosted-authenticate-service.md:42]:
```suggestion
```

**Review by calebdoxsey — CHANGES_REQUESTED** [review]

**Comment by wasaga** [comment]
I think we need describe limitations using hosted authenticate. 

1. only google and email/password authentication is (currently) supported
2. as hosted authenticate is severed from the pomerium installation, it is not possible to refresh session tokens, meaning the session would live for just about an hour, after which users would have to re-authenticate.

**Pull request merged**